app/main/views/verify.py

```diff
@@ -69,83 +67,50 @@ def activate_user(user_id):
     user = User.from_id(user_id)
 
     # This is the login.gov path
-    login_gov_invite_data = redis_client.get(f"service-invite-{user.email_address}")
-    # TODO REMOVE THIS DEBUG
-    print(  # noqa
-        hilite(
-            f"DATA FROM REDIS WITH USER EMAIL {user.email_address}?  DATA: {login_gov_invite_data}"
+    try:
+        login_gov_invite_data = service_api_client.retrieve_service_invite_data(
+            f"service-invite-{user.email_address}"
         )
-    )
-    # END DEBUG
+    except BaseException:  # noqa
+        # We will hit an exception if we can't find invite data,
+        # but that will be the normal sign in use case
+        login_gov_invite_data = None
     if login_gov_invite_data:
-        login_gov_invite_data = json.loads(login_gov_invite_data.decode("utf8"))
+        # Not clear why we have to call json.loads twice, but we do
+        login_gov_invite_data = json.loads(login_gov_invite_data)
```

Comment:
That definitely seems odd, unless it's a JSON string within a JSON string?

Reply:
Oh, that comment is out of date.  When I was sending data back from api, I was calling json.dumps(), but apparently there is an implicit json.dumps() already happening.  I have removed that from the API, so only one json.loads() is needed here.  I will remove the comment.